lib/util.js

```diff
@@ -248,13 +248,27 @@ function removeColorCodes(input) {
   return input.replace(/\x1b\[[0-9;]*m/g, '');
 }
 
+const jestHelpers = {
```

Comment:
can we put it into jest adapter file?

Reply:
no, it creates circular dependency then, already tested it